W-11584150: improved description of dynamic queries caveats (#2043)
mulesoft/docs-connectors: db/1.10/modules/ROOT/pages/database-connector-select.adoc

You can apply input parameters only to parameters in a `WHERE` clause. To modify any other part of the query, use the DataWeave interpolation operator.

> Since the expression values may not be available at design time, using this feature might throw DataWeave evaluation errors or affect the Output Metadata (DataSense) of the Select Operation.

Reviewer: ```suggestion
Because the expression values may not be available at design time, using dynamic queries might throw DataWeave evaluation errors or affect the output metadata (DataSense) of the *Select* operation.
```

> To default to a value (and avoid the aforementioned inconvenience) the default modifier may be used as follows:

Reviewer: ```suggestion
To default to a value and avoid the aforementioned inconvenience, use the default modifier as follows:
